images/windows_ami_sg.tf

```
@@ -0,0 +1,24 @@
+# Security group for manually finishing AMI setup for imported Windows AMIs
+resource "aws_security_group" "windows_ami" {
+  # We include a name for convenience because this Security Group will be
```

Unnecessary capitalization:
```suggestion
  # We include a name for convenience because this security group will be
```

---

Please see 97ee8a1.